apps/fabric-website/src/pages/Overviews/ResourcesPage/docs/default/ResourcesDeveloperResources.md

```diff
@@ -29,16 +29,16 @@ SharePoint uses Fabric, so if you’re building on top of or within a SharePoint
 
 <ul class="md-list--flex">
   <li class="mdut--half">[SharePoint Framework overview](https://aka.ms/spfx)</li>
-  <li class="mdut--half">[Use Fabric components in web parts](https://aka.ms/spfx-fabric-react)</li>
+  <li class="mdut--half">[Use Fluent UI components in web parts](https://aka.ms/spfx-fabric-react)</li>
```

Comment:
For this and the next few instances, should we specify Fluent UI React?

Reply:
```suggestion
  <li class="mdut--half">[Use Fluent UI React components in web parts](https://aka.ms/spfx-fabric-react)</li>
```